src/main/python/org/broadinstitute/hellbender/gcnvkernel/postprocess/segment_quality_utils.py

```diff
@@ -145,7 +145,8 @@ def get_log_constrained_posterior_prob(self,
         """
         assert start_index >= 0
         assert end_index < self.num_sites
-        assert end_index >= start_index
+        assert end_index >= start_index, \
```

Comment:
Thanks, I'm not a fan of using `assert` except for testing, but at least you've added an error message.

Reply:
I copied Mehrtash's style, but I'm open to suggestions.